csrc/attention/attention_kernels.cu

```diff
@@ -782,32 +858,32 @@ void paged_attention_v2_launcher(
     // NOTE(woosuk): To reduce the compilation time, we only compile for the
     // head sizes that we use in the model. However, we can easily extend this
     // to support any head size which is a multiple of 16.
-    case 64:
-      LAUNCH_PAGED_ATTENTION_V2(64);
-      break;
-    case 80:
-      LAUNCH_PAGED_ATTENTION_V2(80);
-      break;
-    case 96:
-      LAUNCH_PAGED_ATTENTION_V2(96);
-      break;
-    case 112:
-      LAUNCH_PAGED_ATTENTION_V2(112);
-      break;
+    // case 64:
+    //   LAUNCH_PAGED_ATTENTION_V2(64);
+    //   break;
+    // case 80:
+    //   LAUNCH_PAGED_ATTENTION_V2(80);
+    //   break;
+    // case 96:
+    //   LAUNCH_PAGED_ATTENTION_V2(96);
+    //   break;
+    // case 112:
+    //   LAUNCH_PAGED_ATTENTION_V2(112);
+    //   break;
     case 128:
       LAUNCH_PAGED_ATTENTION_V2(128);
       break;
-    case 256:
-      LAUNCH_PAGED_ATTENTION_V2(256);
-      break;
+    // case 256:
+    //   LAUNCH_PAGED_ATTENTION_V2(256);
+    //   break;
     default:
```

Comment:
why comment the above cases?

Reply:
We were trying to speed up compiling when testing the code. Comment has been removed now.